sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2928,31 +2928,6 @@ object functions {
   // DateTime functions
   //////////////////////////////////////////////////////////////////////////////////////////////
 
-  /**
-   * (Scala-specific) Creates a datetime interval
-   *
-   * @param years Number of years
-   * @param months Number of months
-   * @param weeks Number of weeks
-   * @param days Number of days
-   * @param hours Number of hours
-   * @param mins Number of mins
-   * @param secs Number of secs
-   * @return A datetime interval
-   * @group datetime_funcs
-   * @since 3.2.0
-   */
-  def make_interval(
```

Comment:
do we have `make_ym_interval` and `make_dt_interval` now?

Reply:
I wouldn't remove until we decide to deprecate `CalendarIntervalType`?

Reply:
Is it better to remove it before a release? `@since 3.2.0`

Reply:
Will we deprecate and remove `CalendarIntervalType`?

Reply:
> do we have make_ym_interval and make_dt_interval now?

@cloud-fan As SQL functions, see https://github.com/apache/spark/pull/32645 and https://github.com/apache/spark/pull/32601 . The functions haven't been exposed to Scala/Python/R APIs yet.

> Will we deprecate and remove CalendarIntervalType?

@HyukjinKwon Yes, we will I hope. The questions is only when.

Reply:
okie then i am good